docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  postgres:
+    container_name: feedback-local-postgres
+    image: postgres:17
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
```

Comment:
[dust] Something for you to consider is storing these in an env file and referencing the env vars instead, but if this is just for local testing, I think this is a valid approach.

That said, you may want to set the `TZ` environment variable so that you don't run into issues with dates and times.